### PR TITLE
frontend: Add base templates

### PR DESCRIPTION
**Pull request by drewwalters96**
Adds base templates other pages can extend.
- Introduces jQuery and Bootstrap
- Adds a `setup.py`

### CODE REVIEW

File: stlcapsmh/stlcapsmh.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                       'Professional Studies.',
+        'path': '/'
+    }
+]
+
+
+@APP.route('/')
+def get_index():
+    """Display the homepage."""
+    # TODO(@drewwalters96): There should be a safer way to get a page from the
```
Comment (annapankiewicz) [inline stlcapsmh/stlcapsmh.py:58]: Agreed

**Review by annapankiewicz — APPROVED** [review]
Looks good! Thanks

**Comment by drewwalters96** [comment]
LGTM. Displaying the footer shouldn't depend on `main.css`, but I don't want to block this. I'll merge and create an issue.

**Comment by drewwalters96** [comment]
[Base Template](https://trello.com/c/0ZEZrDOE/24-base-template)

**Pull request merged**